Issue with deploying to cloud run due to Java version inconsistency

I followed this guide: https://cloud.google.com/run/docs/quickstarts/build-and-deploy/deploy-java-service.  It results in a failed build due to the pom.xml requiring Spring Boot v3 and Java 17.  The cloud run buildpack defaults to Java 11.  


The same issue occurs when clicking the https://deploy.cloud.run link from the README file.  It results in this error:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project microservice-template: Fatal error compiling: error: invalid target release: 17 -> [Help 1]


Would you either downgrade to spring boot that will run on 11 or perhaps include instructions on how to make this run with a java 17 option in the buildpack?

--- a/project.toml
+++ b/project.toml
@@ -1,0 +1,8 @@
+# Default version is Java 11
+#  - See https://cloud.google.com/docs/buildpacks/java#specify_a_java_version
+# Match the version required in pom.xml by setting it here
+#  - See https://cloud.google.com/docs/buildpacks/set-environment-variables#build_the_application_with_environment_variables
+
+[[build.env]]
+  name = "GOOGLE_RUNTIME_VERSION"
+  value = "17"

